packages/react-hooks/src/hooks/media.ts

```diff
@@ -0,0 +1,36 @@
+import {useState, useEffect} from 'react';
+
+export function useMedia<ValueType>(
+  queries: string[],
+  values: ValueType[],
+  defaultValue: ValueType,
+) {
+  const mediaQueryLists = queries.map(query => {
+    return window.matchMedia(query);
```

Comment:
We should return a default value (`false`?) if you are rendering on the server.

```
 if (typeof window === 'undefined') {
    return false;
  }
```

Reply:
`false.addEventListener(...)` below will 💥

Reply:
Sorry I missed that. You can return a noop object matching the signature of what you would expect on the client:

```
const noopMediaQueryList = {
  media: '',
  onchange: noop,
  addListener: noop,
  addEventListener: noop,
  removeListener: noop,
  removeEventListener: noop,
  dispatchEvent: () => false,
  matches: false,
};
```

Reply:
Thanks I totally forgot about SSR!